osbuild/main_cli.py

```diff
@@ -168,6 +168,15 @@ def osbuild_cli():
             if r["success"] and exports:
                 for pid in exports:
                     export(pid, output_directory, object_store, manifest)
+    except OSBuildError as e:
+        if args.json:
+            r = fmt.format_osbuild_error(e)
+            json.dump(r, sys.stdout)
+            sys.stdout.write("\n")
+        else:
```

Comment:
This is not very extensible since for the next error that is not derived from `OSBuildError`, like all of pythons native errors, you need to duplicate the code.